### PR TITLE
Add the transiflex rule in the help

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: CONST_Makefile [hunk 1]
```diff
@@ -261,6 +264,9 @@ help:
 	@echo "- compile-py-catalog		Compile the angular translation catalog"
 	@echo "- flake8					Run flake8 checker on the Python code"
 	@echo "- lint-ngeo				Check the JavaScript code with linters for ngeo"
+ifdef SECOUDARY_HELP
```
Comment (elemoine) [inline CONST_Makefile]: -> SECONDARY

File: geoportailv3.mk [hunk 2]
```diff
@@ -17,6 +17,8 @@ DISABLE_BUILD_RULES = test-packages test-packages-ngeo
 
 CONFIG_VARS += ldap
 
+UTILITY_HELP = "- update-translations		Syncronise the localisation with transifex"
```
Comment (elemoine) [inline geoportailv3.mk]: Synchronize

Comment (elemoine) [inline geoportailv3.mk]: the localisation -> the translations

File: geoportailv3.mk [hunk 3]
```diff
@@ -17,6 +17,8 @@ DISABLE_BUILD_RULES = test-packages test-packages-ngeo
 
 CONFIG_VARS += ldap
 
+UTILITY_HELP = "- update-translations		Syncronise the localisation with Transifex"
```
Comment (elemoine) [inline geoportailv3.mk]: Syncronise the localisation with Transifex -> Synchronize the translations with Transifex

File: CONST_Makefile (hunk 1 above)
Comment (sbrunner) [inline CONST_Makefile]: Oups, thanks :-)

**Comment by jaykayone** [comment]
LGTM :+1:

**Pull request merged**